lib/react_on_rails/configuration.rb

```diff
@@ -64,7 +67,8 @@ def initialize(node_modules_location: nil, server_bundle_js_file: nil, prerender
                    build_production_command: nil,
                    same_bundle_for_client_and_server: nil,
                    i18n_dir: nil, i18n_yml_dir: nil, i18n_output_format: nil,
-                   random_dom_id: nil, server_render_method: nil, rendering_props_extension: nil)
+                   random_dom_id: nil, server_render_method: nil, rendering_props_extension: nil,
+                   components_directory: nil, load_bundle: nil)
```

Comment:
There are 3 places with the same (nearly) set of parameters, but very different orders, so hard to compare. Group related parameters together? Alphabetic order? Again probably not in this PR.